sklearn/decomposition/_dict_learning.py

```diff
@@ -857,6 +890,10 @@ def dict_learning_online(
         Number of iterations run. Returned only if `return_n_iter` is
         set to `True`.
 
+    n_steps : int
+        The number of iteration on data batches that has been
+        performed before. Returned only if `max_iter` is not None.
```

Comment:
Why do we need to return this? For testing only? If a user is interested in this kind of details, I think they should use the object oriented API.

Reply:
no need. I removed it